src/pages/home/report/ReportActionItem.tsx

```diff
@@ -178,7 +178,8 @@ function ReportActionItem({
     const {translate} = useLocalize();
     const {shouldUseNarrowLayout} = useResponsiveLayout();
     const blockedFromConcierge = useBlockedFromConcierge();
-    const originalReportID = useMemo(() => ReportUtils.getOriginalReportID(report.reportID, action) ?? '-1', [report.reportID, action]);
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const originalReportID = useMemo(() => ReportUtils.getOriginalReportID(report.reportID, action) || '-1', [report.reportID, action]);
```

Comment:
There are few other places that we're using `originalReportID ?? '-1'`, do you think we should update them as well?
https://github.com/Expensify/App/blob/6ca91cddfbb4bbcd0ca24c9f10549057b6fd7332/src/libs/actions/ReportActions.ts#L35

https://github.com/Expensify/App/blob/6ca91cddfbb4bbcd0ca24c9f10549057b6fd7332/src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx#L214

Reply:
I think it would be safe to do it, but I think it's best that we have a guideline to use || instead of ?? for string and update them all in one issue (break in several PRs perhaps).

Reply:
Can we just update these 2 places here? And then we can discuss about the second part later.

cc @cristipaval

Reply:
OR, can we update `getOriginalReportID` to return undefined instead empty string?

Reply:
I think returning `reportID || undefined` looks unclean.

I have updated to use `||` for now. We can revert it if @cristipaval disagrees

Reply:
Btw, I'm currently having trouble running the native app.

Reply:
Same here :/ I can't run native too

Reply:
Can we raise this for discussion somewhere?

Reply:
Yeah there was a thread here: https://expensify.slack.com/archives/C01GTK53T8Q/p1724842994690509